src/libs/Navigation/AppNavigator/createCustomFullScreenNavigator/index.tsx

```diff
@@ -72,6 +65,6 @@ function ResponsiveStackNavigator(props: ResponsiveStackNavigatorProps) {
     );
 }
 
-ResponsiveStackNavigator.displayName = 'ResponsiveStackNavigator';
+FullScreenNavigator.displayName = 'FullScreenNavigator';
```

Comment:
```suggestion
CustomFullScreenNavigator.displayName = 'CustomFullScreenNavigator';
```

(same for index.native.tsx)